lesson09/src/db/migrations/init.up.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE groups (
```

Comment:
This is not what is meant by migrations. Migration needs to be applied at the application start. You had a library here, but it was just a separate app, that received it's own requests. You may use that one, but need to integrate it into the main application, and perform migration for the database, based on the predefined setup

Reply:
added migration using migrate from wshaman/migrate pkg, refactored, created migration schemas and integrated into main function.